### ISSUE TITLE
Tcl errors on install

### ISSUE THREAD

**Issue by Spacechild1** [pure-data/deken#238]
Deken 0.7.2, Pd 0.51-4, Windows 7

When I try to install any package to any folder, I get the following Tcl error
```
bad option "select": must be -default, -detail, -icon, -message, -parent, -title, or -type
bad option "select": must be -default, -detail, -icon, -message, -parent, -title, or -type
    while executing
"tk_messageBox -message Please select a (writable) installation directory! -type retrycancel -default retry -icon warning -parent .externals_searchui"
    ("eval" body line 1)
    invoked from within
"eval tk_messageBox ${_args}"
    (procedure "::deken::ensure_installdir" line 26)
    invoked from within
"::deken::ensure_installdir "" ${filename}"
    (procedure "::deken::clicked_link" line 6)
    invoked from within
"::deken::clicked_link http://puredata.info/Members/fjkraan/software/freeverb~/1.2.3/freeverb~-v1.2.3-(Windows-i386-32)(Sources)-externals.zip freeverb..."
    (command bound to event)
```
The folders in question are certainly writeable!

Also, Deken always shows me the folder dialog *first*, which I think should only appear if I click "No" on the "Install ... to ...?" dialog.

Furthermore, when I click "cancel" on the install folder dialog, the "Install ... to ...?" dialog shows "]" as the destination folder. If I then click "Cancel", I get the following Tcl error:

```
couldn't change working directory to "": no such file or directory
couldn't change working directory to "": no such file or directory
    while executing
"cd $installdir"
    (procedure "::deken::utilities::extract" line 4)
    invoked from within
"::deken::utilities::extract $installdir $filename $fullpkgfile $keep"
    (procedure "::deken::install_package" line 37)
    invoked from within
"::deken::install_package ${fullpkgfile} ${filename} ${installdir} ${::deken::keep_package}"
    (procedure "::deken::clicked_link" line 29)
    invoked from within
"::deken::clicked_link http://puredata.info/Members/fjkraan/software/freeverb~/1.2.3/freeverb~-v1.2.3-(Windows-i386-32)(Sources)-externals.zip freeverb..."
    (command bound to event)
```

BTW, in the "Install ... to ...?" dialog, clicking the "X" is apparantely interpreted as "No" instead of "Cancel", so the folder dialog appears.

This is my Pd console output:
```
Loading plugin: C:/Program Files (x86)/Pd/tcl/pd_deken.tcl
Loading plugin: C:/Program Files (x86)/Pd/tcl/pd_docsdir.tcl
Loading plugin: C:/Users/Christof/AppData/Roaming/Pd/deken-plugin/deken-plugin.tcl
[deken] installed version [0.5.0] < 0.7.2...overwriting!
[deken] deken-plugin.tcl (Pd externals search) loaded from C:/Users/Christof/AppData/Roaming/Pd/deken-plugin.
[deken] Start searching for externals... freeverb~
Commencing downloading of:
http://puredata.info/Members/fjkraan/software/freeverb~/1.2.3/freeverb~-v1.2.3-(Windows-i386-32)(Sources)-externals.zip
Into ...
```

**Comment by umlaeute** [comment]
could you please test the latest and greatest (currently only available via `git`)?

**Comment by Spacechild1** [comment]
All of the listed issues are resolved with 0.7.3. Thanks!
